src/protocol/graphics.ts

```diff
@@ -239,19 +239,25 @@ export function setupGraphics(store: UIStore) {
     assert(pause);
 
     const rv = await pause.repaintGraphics();
-    if (pause === ThreadFront.currentPause && rv) {
-      const { mouse } = await getGraphicsAtTime(ThreadFront.currentTime);
-      let { description, screenShot } = rv;
-      if (screenShot) {
-        repaintedScreenshots.set(description.hash, screenShot);
-      } else {
-        screenShot = repaintedScreenshots.get(description.hash);
-        if (!screenShot) {
-          console.error("Missing repainted screenshot", description);
-          return;
+    if (pause === ThreadFront.currentPause) {
+      if (rv) {
+        const { mouse } = await getGraphicsAtTime(ThreadFront.currentTime);
+        let { description, screenShot } = rv;
+        if (screenShot) {
+          repaintedScreenshots.set(description.hash, screenShot);
+        } else {
+          screenShot = repaintedScreenshots.get(description.hash);
+          if (!screenShot) {
+            console.error("Missing repainted screenshot", description);
+            return;
+          }
         }
+        paintGraphics(screenShot, mouse);
+      } else if (rv === null) {
```

Comment:
This can just be an "else" branch, there aren't other meaningful falsey values and when repainting we always want to paint *something* if we can.